app/models/organisation.rb

```diff
@@ -46,6 +46,10 @@ def local_authority?
     local_authority_code.present?
   end
 
+  def trust_or_la?
+    trust? || local_authority?
```

Comment:
This is identical to `school_group?`

Reply:
I have no idea what the intent behind all these is, but in terms of the code/DB they're different.

`trust?` and `local_authority?` are based on the presence of strings whereas `school_group?` checks for a subclass (which is unrelated to those two values).